{{ cookiecutter.__package_name_kebab_case }}/Dockerfile

```diff
@@ -119,7 +119,7 @@ RUN ln -s /run/secrets/poetry-auth /home/app/.config/pypoetry/auth.toml
 FROM base AS app
 
 # Copy the package source code to the working directory.
-COPY . .
+COPY --chown=app:app . .
```

Comment:
Could you also add this change to the other `COPY` commands (I believe there are a few others above)? Thanks!